plugins/mac-virtualcam/src/camera-extension/cmake/macos/Info.plist.in

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleName</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
+    <key>CFBundleVersion</key>
+    <string>${MACOSX_BUNDLE_BUNDLE_VERSION}</string>
+    <key>CFBundleShortVersionString</key>
+    <string>${MACOSX_BUNDLE_SHORT_VERSION_STRING}</string>
+    <key>CFBundleExecutable</key>
+    <string>com.obsproject.obs-studio.mac-camera-extension</string>
```

Comment:
These strings still say `mac-camera-extension`.

Reply:
That is indeed on purpose.